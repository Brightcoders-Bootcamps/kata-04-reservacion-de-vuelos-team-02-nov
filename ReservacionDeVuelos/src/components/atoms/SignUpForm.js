import React from 'react';
import {Text, TextInput, View} from 'react-native';
import {styles} from '../../styles/SignUpStyles';
import CheckBox from '@react-native-community/checkbox';

const SignUpForm = () => {
  return (
    <View>
      <Text style={styles.formText}>First Name</Text>
      <TextInput style={styles.textInput} />
      <Text style={styles.formText}>Email *</Text>
      <TextInput style={styles.textInput} />
      <Text style={styles.formText}>Password *</Text>
      <TextInput style={styles.textInput} secureTextEntry={true} />
      <Text style={styles.characters}>
        Use 8 or more characters with a mix of letters, numbers, and sumbols
      </Text>
      <View style={styles.checkboxFather}>
        <View style={styles.checkBoxContainer}>
          <CheckBox />
          <Text>I agree to the Trems and Privacy Policy *</Text>
        </View>
        <View style={styles.checkBoxContainer}>
          <CheckBox />
          <Text>Subscribe for select product updates.</Text>
        </View>
      </View>
    </View>
  );
};

export default SignUpForm;
