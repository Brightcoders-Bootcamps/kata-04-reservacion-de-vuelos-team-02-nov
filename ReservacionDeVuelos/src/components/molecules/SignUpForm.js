import React from 'react';
import {View} from 'react-native';
import {styles} from '../../styles/SignUpStyles';
import {SignUpConstants} from '../../utils/Constants';
import SignUpFormInput from '../atoms/SignUpAtoms/SignUpFormInput';
import SignUpFormCheckBox from '../atoms/SignUpAtoms/SignUpFormCheckBox';
const SignUpForm = () => {
  return (
    <View>
      <SignUpFormInput inputName={SignUpConstants.NameInputTitle} />
      <SignUpFormInput inputName={SignUpConstants.EmailInputTitle} />
      <SignUpFormInput inputName={SignUpConstants.PasswordInputTitle} />
      <View style={styles.checkboxFather}>
        <View style={styles.checkBoxContainer}>
          <SignUpFormCheckBox
            checkboxMessage={SignUpConstants.TermsCheckboxTitle}
          />
        </View>
        <View style={styles.checkBoxContainer}>
          <SignUpFormCheckBox
            checkboxMessage={SignUpConstants.UpdatesCheckboxTitle}
          />
        </View>
      </View>
    </View>
  );
};

export default SignUpForm;
