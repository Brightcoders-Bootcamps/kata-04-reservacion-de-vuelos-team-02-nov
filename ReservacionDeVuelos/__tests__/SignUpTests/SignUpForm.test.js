import React from 'react';
import SignUpForm from '../../src/components/molecules/SignUpForm';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<SignUpForm />).toJSON();
  expect(tree).toMatchSnapshot();
});
