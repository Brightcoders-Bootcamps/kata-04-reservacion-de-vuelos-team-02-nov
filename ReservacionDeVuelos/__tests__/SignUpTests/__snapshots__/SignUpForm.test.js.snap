// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correclty snapshot 1`] = `ShallowWrapper {}`;
